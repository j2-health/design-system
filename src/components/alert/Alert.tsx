import { Alert as AntdAlert, AlertProps } from 'antd'

type Props = Expand<AlertProps>

const Alert = (props: Props) => {
  return <AntdAlert {...props} />
}

export { Alert }
