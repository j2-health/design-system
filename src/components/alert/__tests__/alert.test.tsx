import { render } from '@testing-library/react'
import { Alert } from '../Alert'

describe('Alert', () => {
  it('should render correctly', () => {
    const { container } = render(<Alert message="Hello world" type="success" />)
    expect(container).toMatchSnapshot()
  })
})
