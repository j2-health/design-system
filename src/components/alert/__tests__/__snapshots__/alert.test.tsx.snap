// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should render correctly 1`] = `
<div>
  <div
    class="ant-alert ant-alert-info ant-alert-no-icon css-dev-only-do-not-override-ccdg5a"
    data-show="true"
    role="alert"
  >
    <div
      class="ant-alert-content"
    >
      <div
        class="ant-alert-message"
      >
        Hello world
      </div>
    </div>
  </div>
</div>
`;
